### PR TITLE
mgmt, redisenterprise, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/redisenterprise/azure-resourcemanager-redisenterprise/src/test/java/com/azure/resourcemanager/redisenterprise/RedisEnterpriseManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            // @embedmeStart
+            cluster = redisEnterpriseManager.redisEnterprises()
+                .define(clusterName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withSku(new Sku().withName(SkuName.ENTERPRISE_E10).withCapacity(2))
+                .withIdentity(new ManagedServiceIdentity().withType(ManagedServiceIdentityType.NONE))
+                .withMinimumTlsVersion(TlsVersion.ONE_TWO)
+                .create();
+            // @embedEnd
```
Comment (weidongxu-microsoft) [inline sdk/redisenterprise/azure-resourcemanager-redisenterprise/src/test/java/com/azure/resourcemanager/redisenterprise/RedisEnterpriseManagerTests.java:83]: `@embedmeEnd`

If there was difference in your previous PR, please fix them as well.

They would be rendered here https://github.com/azure/azure-sdk-for-java/tree/main/sdk/databricks/azure-resourcemanager-databricks#examples

Comment (v-hongli1) [reply sdk/redisenterprise/azure-resourcemanager-redisenterprise/src/test/java/com/azure/resourcemanager/redisenterprise/RedisEnterpriseManagerTests.java]: Fixed in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**